templates/addons/calico.yaml

```diff
@@ -1,5 +1,1088 @@
 apiVersion: apiextensions.k8s.io/v1
 kind: CustomResourceDefinition
+metadata:
```

Comment:
Are these diffs just generated templates from the new Calico source? And same with all the deletions in the other folders of `addons`?

Reply:
Yes, this is automated via the make file command where it grabs the latest release then extracts it.